CMakeLists.txt

```diff
@@ -551,8 +551,8 @@ if (WIN32)
     "${DDK_ROOT}/tools/tracing/${DDK_SFX}/dbghelp.dll"
     "${PROGFILES}/Microsoft Visual Studio */Common7/IDE/Remote Debugger/${ARCH_SFX}/dbghelp.dll"
     "${PROGFILES32}/Windows Kits/*/Debuggers/${ARCH_SFX}/dbghelp.dll"
-    # For Visual Studio 2010+, x64 dbghelp.dll resides in Program
-    # Files (x86) rather than Program Files.
+    # In case if SDK is not installed and we have Visual Studio, dbghelp.dll may be found in the
+    # Visual Studio's directory in subfolder Remote Debugger/x86 or /x64 folders (xref i#1956).
```

Comment:
This same subpath is at line 553 right above.  Let's put them adjacent to each other (otherwise the comment is weird only being here).  It looks like only old VS (2005, 2008) put dbghelp into Remote Debugger in 64-bit progfiles and everything since has it in 32-bit progfiles, and the regression must have been to set PROGFILES to the 64-bit path for a 32-bit config.  So the comment should also explain why we need two different paths, one for each progfiles path.